src/main/java/com/snowplowanalytics/redash/RedashClient.java

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright (c) 2018 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.redash;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.snowplowanalytics.redash.model.BaseEntity;
+import com.snowplowanalytics.redash.model.User;
+import com.snowplowanalytics.redash.model.UserGroup;
+import com.snowplowanalytics.redash.model.datasource.DataSource;
+import okhttp3.*;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.*;
+
+public class RedashClient {
+
+    private static final String APIKEY_STRING = "?api_key=";
+    private static final String API_STRING = "/api";
```

Comment:
Please rename this (and the other URL constants below) to something like  `*_PREFIX` or `*_URL_PREFIX`